src/OrchardCore.Modules/OrchardCore.Apis.GraphQL/Startup.cs

```diff
@@ -72,7 +74,7 @@ public override void ConfigureServices(IServiceCollection services)
                     User = ctx.User,
                 };
                 c.ExposeExceptions = exposeExceptions;
-                c.MaxDepth = configuration.GetValue<int?>($"OrchardCore_Apis_GraphQL:{nameof(GraphQLSettings.MaxDepth)}") ?? 20;
+                c.MaxDepth = configuration.GetValue<int?>($"OrchardCore_Apis_GraphQL:{nameof(GraphQLSettings.MaxDepth)}") ?? 100;
```

Comment:
fix `Query is too nested to execute. Depth is 59 levels, maximum allowed on this endpoint is 20.`